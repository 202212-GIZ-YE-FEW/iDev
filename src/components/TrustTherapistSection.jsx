import Image from "next/image";
import Link from "next/link";
import { withTranslation } from "next-i18next";

import PageIntro from "./PageIntro";
import Button from "./ui/Button";
function TrustTherapists({ t }) {
    return (
        <>
            <div className='container flex flex-col space-y-8 justify-between'>
                <PageIntro title={t("trustTherapistTitle")} />
                <Image
                    src='/trust_therapists.png'
                    alt='trust_therapist'
                    width={600}
                    height={300}
                />
                <p className='pb-20'>{t("trustTherapistSubtitle")}.</p>
                <Link
                    className='w-fit xl:w-full rtl:w-full'
                    href='/appointment'
                >
                    <Button
                        content={t("bookAppointment")}
                        textTransform='uppercase'
                        filled='true'
                        size='large'
                        fontSize='text-lg md:text-xl lg:text-2xl'
                        radius='md'
                    />
                </Link>
            </div>
        </>
    );
}
export default withTranslation("home")(TrustTherapists);
