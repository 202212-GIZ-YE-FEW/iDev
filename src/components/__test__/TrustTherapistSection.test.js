import renderer from "react-test-renderer";

import TrustTherapistSection from "../TrustTherapistSection";

it("renders correctly", () => {
    const tree = renderer.create(<TrustTherapistSection />).toJSON();
    expect(tree).toMatchSnapshot();
});
