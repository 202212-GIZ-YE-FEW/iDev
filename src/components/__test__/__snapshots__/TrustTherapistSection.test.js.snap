// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container flex flex-col space-y-8 justify-between"
>
  <div
    className="mb-5"
  >
    <span
      className="font-normal block text-3xl md:text-4xl rtl:md:text-3xl lg:text-4xl rtl:lg:text-4xl uppercase break-words"
    >
      trustTherapistTitle
    </span>
  </div>
  <img
    alt="trust_therapist"
    data-nimg="1"
    decoding="async"
    height={300}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Ftrust_therapists.png&w=1200&q=75"
    srcSet="/_next/image?url=%2Ftrust_therapists.png&w=640&q=75 1x, /_next/image?url=%2Ftrust_therapists.png&w=1200&q=75 2x"
    style={
      Object {
        "color": "transparent",
      }
    }
    width={600}
  />
  <p
    className="pb-20"
  >
    trustTherapistSubtitle
    .
  </p>
  <div
    className="w-fit"
  >
    <button
      className="font-normal whitespace-nowrap cursor-pointer  capitalize text-lg md:text-xl lg:text-2xl rounded-md px-[25px] md:px-[35px] py-[8px] font-medium bg-cyan text-black "
    >
      bookAppointment
    </button>
  </div>
</div>
`;
